src/net/sourceforge/kolmafia/persistence/CoinmastersDatabase.java

```diff
@@ -251,8 +260,10 @@ public static final int getPrice(final int itemId, final Map<Integer, Integer> p
 
   public static final void clearPurchaseRequests(CoinmasterData data) {
     // Clear all purchase requests for a particular Coin Master
-    COINMASTER_ITEMS.values().removeIf(request -> request.getData() == data);
-    COINMASTER_ROWS.values().removeIf(request -> request.getData() == data);
+    for (List<CoinMasterPurchaseRequest> list : COINMASTER_ITEMS.values()) {
+      list.removeIf(request -> request.getData() == data);
+    }
+    COINMASTER_ROWS.values();
```

Comment:
does nothing?

Reply:
Ah. I cut (rather than copied) the .removeIf and moved it.
Good catch.